### DISCUSSION TITLE
Cronicle - V2 Timeline

### DISCUSSION THREAD

**Discussion by pradip-allwyn** [jhuckaby/Cronicle#768] [Q&A] [answered]
Hi, Thanks for the Awsome tool, and thank you for working on V2.

are there any details on what are the features that you are working on V2, as a list of features would be good to see.

 what would be the tentative timeline?  no pressure :)

**Comment by jhuckaby** [comment]
Hi there!  Happy to answer, but I'm going to move this to a discussion, as this isn't really an "issue".

**Comment by jhuckaby** [comment, marked as answer]
This discussion thread has a good summary of v2 features: https://github.com/jhuckaby/Cronicle/discussions/453

I apologize for lack of a better presentation of these.  I am working on the marketing website, which will eventually go live at `orchestra.dev`, but it's not ready yet.

For a timeline, I'm afraid I must apologize again.  I feel absolutely terrible, because I've been answering this question for the last 3 years, giving answers such as "sometime in 2022", "sometime in 2023", and "sometime in 2024".  These statements were all truthful at the time -- I really thought I could finish Orchestra by 2022, then by 2023, and now by 2024, but life just keeps getting in the way, and I often have to take week-long or sometimes month-long breaks.

I feel more and more awful each time this question comes up, because I'm afraid that whatever I say, I'll be lying, and I feel more and more pressure to finish by the new date I give out.

I will say this:  Orchestra is my [magnum opus](https://en.wikipedia.org/wiki/Masterpiece).  I will not stop until it is finished, come hell or high water, but... perfection takes time 😊 

I want to say that I'll finish it by the end of 2024.  I really hope I can make that date.  But I really can't promise anything at this point.  My day job takes 90% of my time and energy, and my family takes the other 10%.  I'm left with so very little energy to devote to Orchestra.  But I'm trying -- really, really hard.

**Comment by jhuckaby** [comment]
Happy 2026! 🎉 

Sorry for all the huge delays.  I quit my job in 2025 to finally get it finished, and I did it -- just barely 😝 

And here is the beta of xyOps (Cronicle v2) ready for testing:

https://github.com/pixlcore/xyops

Blog post with more info:

https://pixlcore.com/blog/dev-update-2026-01

Let me know what you think!

❤️

**Reply by pablos123** [reply]
Hi,

First of all: awesome tool!

I'm here to ask if there is a plan to create a migration process from Cronicle v1 to Cronicle V2, seeing that now is xyOps (I found out today).

Sorry if this is not the place to ask.

Greetings!

**Reply by jhuckaby** [reply]
You betcha: https://github.com/pixlcore/xyops/blob/main/docs/cronicle.md

**Comment by cju-mt** [comment]
I think a lot of us Cronicle addicts would like to know that 😊

Thanks,
Dave

**Reply by jhuckaby** [reply]
Yup!  Here you go: https://github.com/pixlcore/xyops/blob/main/docs/cronicle.md
